lib/home/widgets/task_list.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+
+class TaskCard extends StatelessWidget {
+  const TaskCard(
+    this.task,
+    {
+    super.key,
+  });
+
+  final String task;
```

Comment:
instead of String, try to put there Task - https://github.com/kamilgadek/to_do_list_kamilg/blob/main/packages/tasks_repository/lib/src/models/task.dart